CacheAdvance.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name     = 'CacheAdvance'
-  s.version  = '0.3.0'
+  s.version  = '1.0.0'
```

Comment:
This field is [required](https://guides.cocoapods.org/syntax/podspec.html#version).